libpod/runtime.go

```diff
@@ -379,7 +383,7 @@ func makeRuntime(ctx context.Context, runtime *Runtime) (err error) {
 		json := supportsJSON[name]
 		nocgroups := supportsNoCgroups[name]
 
-		ociRuntime, err := newConmonOCIRuntime(name, paths, runtime.conmonPath, runtime.config, json, nocgroups)
+		ociRuntime, err := newConmonOCIRuntime(name, paths, runtime.conmonPath, runtime.config, json, nocgroups, supportsKVM[name])
```

Comment:
I think we should save the value of this like we do nocgroups and json above

Reply:
Removed and figured it out inside of the function.